graders/c/cgrader/cgrader.py

```diff
@@ -647,7 +647,7 @@ def run_check_suite(
             )
             raise UngradableError() from exc
         except et.ParseError as exc:
-            self.result["message"] += f"Error parsing test suite log.\n\n{e}\n"
+            self.result["message"] += f"Error parsing test suite log.\n\n{exc}\n"
```

Comment:
This is a bug introduced by another ruff PR; and should be fixed as soon as possible!